scripts/exglobal_cleanup.sh

```diff
@@ -59,10 +59,11 @@ for (( current_date=first_date; current_date <= last_date; \
         # TODO: This needs to be revamped to not look at the rocoto log.
         # shellcheck disable=SC2312
         if [[ $(tail -n 1 "${rocotolog}") =~ "This cycle is complete: Success" ]]; then
-            YMD="${current_PDY}" HH="${current_cyc}" declare_from_tmpl COM_TOP
-            if [[ -d "${COM_TOP}" ]]; then
+            YMD="${current_PDY}" HH="${current_cyc}" declare_from_tmpl \
+		    COMOUT_TOP:COM_TOP_TMPL
```

Comment:
Line continuations should be indented
```suggestion
		        COMOUT_TOP:COM_TOP_TMPL
```